packages/frontend/app/src/common/sessionsLocalStorage.ts

```diff
@@ -0,0 +1,7 @@
+export const saveItemValue = (key: number, value: boolean) => {
```

Comment:
I would love a more expressive name, like setInterest. Also you'll get the symmetry with checkInterest in naming.